ibacm/src/acm.c

```diff
@@ -896,7 +896,7 @@ static struct acmc_ep *acm_get_ep(int index, uint8_t port_num)
 			if (dev->port[i].state != IBV_PORT_ACTIVE)
 				continue;
 			list_for_each(&dev->port[i].ep_list, ep, entry) {
-				if (index == inx)
+				if (index == inx++)
```

Comment:
Nice catch,
But please can put "inx++" after this "if (index) return ep;" so it will be less error prone.

Reply:
That would look ugly, but resemble the original for loop better. But since the needless brackets are there, I can use them for something ;-)

Reply:
v2 ready.